### PR TITLE
UA: Test Track: updates

### PR DESCRIPTION
**Pull request by LesiaPavlenko**

### CODE REVIEW

File: packages/UsageAnalysis/files/Test Track/General/Api samples.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Since the API examples' automated tests are currently disabled, we need to verify them manually.
+1. Go Apps > TestManager. Press 'Run' button.
+2. Run "Api Samples".
+3. Expected result: all tests are passed successfully without errors in console.
```
Comment (skalkin) [inline packages/UsageAnalysis/files/Test Track/General/Api samples.md:4]: I don't think we can expect all Api Samples ran in the test manager to just work. Does it work this way?

File: packages/UsageAnalysis/files/Test Track/Viewers/Calendar.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+### Calendar viewer
+
+1. Open the **demog** dataset.
+2. Click the **Add viewer** icon, find **Calendar** viewer and press on it. 
+* Expected result: **Calendar** viewer opens without errors in console.
```
Comment (skalkin) [inline packages/UsageAnalysis/files/Test Track/Viewers/Calendar.md:5]: > Expected result: **Calendar** viewer opens without errors in console. 

Do we really need to write "expected results" for each step? I think it's safe to assume that "no errors should occur" or "no errors in console" should be true for each step?

**Review by skalkin — APPROVED** [review]
I like it! OK to merge but please take into account my feedback as well.

File: packages/UsageAnalysis/files/Test Track/Viewers/Calendar.md (hunk 2 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Calendar.md:5]: Yes, you are right that "no errors should occur" is the expected behavior for all steps of all test scenarios. Moving forward, where possible, I'll place the "expected results" only at the end.

File: packages/UsageAnalysis/files/Test Track/General/Files Cache.md [hunk 3]
```diff
@@ -6,4 +6,9 @@
 6. Rename file to 'test1.txt'
 7. Rename folder to "Folder cache test1"
 8. Delete "Folder cache test1" folder. Make sure the folder is deleted.
-9. Check cache mappings, the mapping for created folder should be deleted also (Context menu, Cache...)
\ No newline at end of file
+9. Check cache mappings, the mapping for created folder should be deleted also (Context menu, Cache...)
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/General/Files Cache.md:9]: Should we validate that the cache works? Not just the settings

File: packages/UsageAnalysis/files/Test Track/General/Molecule in exported CSV.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+### Molecule column in exported CSV
+1. Open the Dataset: Open the SPGI dataset.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/General/Molecule in exported CSV.md:2]: Link the SPGI dataset to the test case to ease the process

File: packages/UsageAnalysis/files/Test Track/Viewers/3d Scatter plot.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+### 3d Scatter plot
+
+1. Open the **demog dataset**.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/3d Scatter plot.md:3]: Link the dataset to test case

File: packages/UsageAnalysis/files/Test Track/Viewers/Calendar.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+### Calendar viewer
+
+1. Open the **demog** dataset.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Calendar.md:3]: Link dataset to the test case

File: packages/UsageAnalysis/files/Test Track/Viewers/Filter panel/Filter with hidden columns.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+### Verify common Hidden Columns Behavior in the Filter Panel
+Ensure that columns are correctly shown or hidden in the filter panel based on their visibility settings.
+* Preconditions: SPGI dataset is loaded into the platform.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Filter panel/Filter with hidden columns.md:3]: Open dataset using Samples dataset icon is better here to give a hint hoe to do it

File: packages/UsageAnalysis/files/Test Track/Viewers/Correlation plot.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+### Correlation plot
+
+1. Open the **SPGI dataset**.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Correlation plot.md:3]: specify how

File: packages/UsageAnalysis/files/Test Track/Viewers/Heatmap.md [hunk 9]
```diff
@@ -9,8 +9,22 @@
...
+* Switch on 'Is heatmap' option in the **Property Pane**. 
+* **Expected Results**: 
+  * After applying the layout, the Heatmap visualization should be identical to the view before the 'Is heatmap' option was switched off. 
+  * The scrollbar position should be saved for layout
+  * There should be no visible differences in the Heatmap display.
+
 ---
 {
   "order": 7
+  "datasets": ["System:DemoFiles/SPGI.csv"]
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Heatmap.md:29]: Link all datasets to the test case

File: packages/UsageAnalysis/files/Test Track/Viewers/Heatmap.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 1. Open SPGI, SPGI-linked1, SPGI-linked2.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Heatmap.md:1]: Open using sample datasets

File: packages/UsageAnalysis/files/Test Track/Viewers/Grid.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
+### Grid viewer
+
+1. Open the **demog** dataset.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Grid.md:3]: Link dataset to the test case

File: packages/UsageAnalysis/files/Test Track/Viewers/Matrix plot.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
+### Matrix plot
+
+1. Open the **SPGI dataset**.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Matrix plot.md:3]: specify how

File: packages/UsageAnalysis/files/Test Track/Viewers/Network diagram.md [hunk 13]
```diff
@@ -0,0 +1,14 @@
+### Network diagram
+
+1. Open the **demog** dataset.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Network diagram.md:3]: Link dataset to the test case

File: packages/UsageAnalysis/files/Test Track/Viewers/Pivot table.md [hunk 14]
```diff
@@ -1,9 +1,31 @@
-Check tickets:
+### Pivot table
 
-* [#2606](https://github.com/datagrok-ai/public/issues/2606)
-* [#2198](https://github.com/datagrok-ai/public/issues/2198)
-* [#2497](https://github.com/datagrok-ai/public/issues/2497)
-* [#2535](https://github.com/datagrok-ai/public/issues/2535)
+1. Open the **demog** dataset.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Pivot table.md:3]: Link dataset to the test case

File: packages/UsageAnalysis/files/Test Track/Viewers/Statistics viewer.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
+### Statistics viewer
+
+1. Open the **demog** dataset.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Statistics viewer.md:3]: Link dataset to the test case

File: packages/UsageAnalysis/files/Test Track/Viewers/Tree map viewer.md [hunk 16]
```diff
@@ -0,0 +1,14 @@
+### Tree map viewer
+
+1. Open the **demog** dataset.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Tree map viewer.md:3]: Link dataset to the test case

File: packages/UsageAnalysis/files/Test Track/Viewers/Trellis plot.md [hunk 17]
```diff
@@ -1,3 +1,51 @@
+### Trellis plot
+
+1. Open the **demog** dataset.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/Trellis plot.md:3]: Link dataset to the test case

File: packages/UsageAnalysis/files/Test Track/Viewers/World cloud.md [hunk 18]
```diff
@@ -0,0 +1,17 @@
+### World cloud
+
+1. Open the **SPGI** dataset.
```
Comment (dnillovna) [inline packages/UsageAnalysis/files/Test Track/Viewers/World cloud.md:3]: Link dataset to the test case

**Review by dnillovna — CHANGES_REQUESTED** [review]

File: packages/UsageAnalysis/files/Test Track/General/Api samples.md (hunk 1 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/General/Api samples.md]: Changed to more appropriate

File: packages/UsageAnalysis/files/Test Track/General/Molecule in exported CSV.md (hunk 4 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/General/Molecule in exported CSV.md:2]: added

File: packages/UsageAnalysis/files/Test Track/Viewers/3d Scatter plot.md (hunk 5 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/3d Scatter plot.md:3]: added "datasets": ["System:DemoFiles/demog.csv"]

File: packages/UsageAnalysis/files/Test Track/Viewers/Correlation plot.md (hunk 8 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Correlation plot.md:3]: done
Open the **SPGI dataset** by pressing 'star' icon in TestTrack (with tooltip "Open test data").

File: packages/UsageAnalysis/files/Test Track/Viewers/Grid.md (hunk 11 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Grid.md:3]: done

File: packages/UsageAnalysis/files/Test Track/Viewers/Network diagram.md (hunk 13 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Network diagram.md:3]: done
  "datasets": ["System:DemoFiles/demog.csv"]

File: packages/UsageAnalysis/files/Test Track/Viewers/Pivot table.md (hunk 14 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Pivot table.md:3]: added   "datasets": ["System:DemoFiles/demog.csv"]

File: packages/UsageAnalysis/files/Test Track/Viewers/Tree map viewer.md (hunk 16 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Tree map viewer.md:3]: added   "datasets": ["System:DemoFiles/demog.csv"]

File: packages/UsageAnalysis/files/Test Track/Viewers/Trellis plot.md (hunk 17 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Trellis plot.md:3]: added     "datasets": ["System:DemoFiles/demog.csv","System:DemoFiles/SPGI.csv"]

File: packages/UsageAnalysis/files/Test Track/Viewers/World cloud.md (hunk 18 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/World cloud.md:3]: added   "datasets": ["System:DemoFiles/SPGI.csv"]

File: packages/UsageAnalysis/files/Test Track/Viewers/Filter panel/Filter with hidden columns.md (hunk 7 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Filter panel/Filter with hidden columns.md:3]: Agree. Changed

File: packages/UsageAnalysis/files/Test Track/General/Files Cache.md (hunk 3 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/General/Files Cache.md:9]: We validate that the cache works by auto tests

File: packages/UsageAnalysis/files/Test Track/Viewers/Matrix plot.md (hunk 12 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Matrix plot.md:3]: done

File: packages/UsageAnalysis/files/Test Track/Viewers/Statistics viewer.md (hunk 15 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Statistics viewer.md:3]: done

File: packages/UsageAnalysis/files/Test Track/Viewers/Heatmap.md (hunk 9 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Heatmap.md:29]: added  "datasets": ["System:DemoFiles/SPGI.csv","System:DemoFiles/SPGI-linked1.csv","System:DemoFiles/SPGI-linked2.csv"]

File: packages/UsageAnalysis/files/Test Track/Viewers/Heatmap.md (hunk 10 above)
Comment (LesiaPavlenko) [reply packages/UsageAnalysis/files/Test Track/Viewers/Heatmap.md:1]: done

**Pull request merged**